benchmark/Streamly/Benchmark/Data/Array/Foreign/Mut.hs

```diff
@@ -89,6 +280,13 @@ main = do
         return (marr, indices)
 
     allBenchmarks array value =
-        [ bgroup (o_1_space_prefix moduleName) $
-            o_1_space_serial_marray value array
+        [ bgroup (o_1_space_prefix moduleName)
+              $ concat
+                    [ o_1_space_serial_marray value array
+                    , o_1_space_generation value
+                    , o_1_space_elimination value
+                    , o_n_heap_serial value
```

Comment:
Why is `o_n_heap` added under `o_1_space_prefix`? The benchmarks may fail to run when run under restricted memory.